src/Estimators/EstimatorManagerCrowd.cpp

```diff
@@ -31,10 +31,25 @@ EstimatorManagerCrowd::EstimatorManagerCrowd(EstimatorManagerNew& em)
   for (int i = 0; i < em.Estimators.size(); i++)
     scalar_estimators_.push_back(em.Estimators[i]->clone());
   MainEstimator = scalar_estimators_[EstimatorMap[MainEstimatorName]];
-  if (em.Collectables)
-    Collectables = em.Collectables->clone();
+  for(UPtr<OperatorEstBase>& upeb : em.operator_ests_)
+  {
+    operator_ests_.emplace_back(upeb->clone());
+  }
 }
 
+void EstimatorManagerCrowd::accumulate(int global_walkers, RefVector<MCPWalker>& walkers, RefVector<ParticleSet>& psets)
```

Comment:
Moved from header.  Implementation code should not be in the header unless it is a huge issue to put in in the .cpp.